src/pages/home/report/ReportActionsView.js

```diff
@@ -201,10 +201,7 @@ class ReportActionsView extends React.Component {
 
     componentDidUpdate(prevProps) {
         if (prevProps.network.isOffline && !this.props.network.isOffline) {
-            if (this.getIsReportFullyVisible()) {
```

Comment:
I think we still want this.

Reply:
so just remove the fetchData bellow?
I tought it was weird to have a openReport in case is fully visible, and fetch data if it's not

Reply:
I see. Well, with this removed we would be marking the report as read if it's not in view. I'm now unsure if `OpenReport` is the best command name to use in this case then. 

If sometimes we want to fetch the report actions + mark the report as read and other times we just want to fetch the report actions - then these seem like two separate actions that we should have two separate commands for as per the 1:1:1 API design philosophy.

Maybe it is best to leave the `fetchData()` call as is for now and we can do something like this in a follow up:

- Create a `ReconnectToReport` command
- Have this command fetch the latest history without marking the report as read

Eventually we would change this to

```js
if (this.getIsReportFullyVisible()) {
    Report.openReport(this.props.reportID);
} else {
    Report.reconnect(this.props.reportID);
}
```

Reply:
should I create a GH for this?
I'll get fetchData back in 😅

Reply:
Yeah I think we will need a follow up for this.

Reply:
Mostly since there is no version of "fetch data" that doesn't mark the report as read now.